vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/textfield/TextAreaWrap.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.flow.component.textfield;
+
+import org.slf4j.LoggerFactory;
+
+import com.vaadin.testbench.unit.ComponentWrap;
+import com.vaadin.testbench.unit.Wraps;
+
+/**
+ * Test wrapper for TextArea components.
+ *
+ * @param <T>
+ *            component type
+ */
+@Wraps(TextArea.class)
+public class TextAreaWrap<T extends TextArea> extends ComponentWrap<T> {
+    /**
+     * Wrap given component for testing.
+     *
+     * @param component
+     *            target component
+     */
+    public TextAreaWrap(T component) {
+        super(component);
+    }
+
+    /**
+     * Set the value to the component if it is usable.
+     * <p>
+     * For a non interactable component an IllegalStateException will be thrown
+     * as the end user would not be able to set a value.
+     *
+     * @param value
+     *            value to set
+     */
+    public void setValue(String value) {
+        ensureComponentIsUsable();
+
+        if (hasValidation() && value != null
```

Comment:
we can maybe have `value != null` as first condition, so that in that case we can avoid reflection usage

Reply:
done